docker/advanced_settings.md

```diff
@@ -129,29 +110,39 @@ Interact with the container. This will open a shell prompt inside the container.
 docker attach mms
 ```
 
-Run the MMS Docker image:
+Run the MMS Docker image without starting the Model Server:
 ```bash
-docker run -itd --name mms -p 80:8080 awsdeeplearningteam/mms_cpu
+docker run -itd --name mms -p 80:8080 -p 81:8081 awsdeeplearningteam/mms_cpu /bin/bash
 ```
 
 Start MMS in the Docker container (CPU config):
 ```bash
-docker exec mms bash -c "mxnet-model-server start --mms-config /models/mms_app_cpu.conf"
+docker exec mms bash -c mxnet-model-server --start --mms-config /home/model-server/config.properties
 ```
 
-Start MMS in the Docker container (GPU config):
+Start MMS in the Docker container using nvidia-docker command as follows. :
 ```bash
-nvidia-docker exec mms bash -c "mxnet-model-server start --mms-config /models/mms_app_cpu.conf"
+nvidia-docker exec mms bash -c mxnet-model-server --start --mms-config /home/model-server/config.properties
+```
+
+**Note**: To use GPU configuration, modify the config.properties to reflect that the model-server should use GPUs.
+
+```yaml
```

Comment:
We should use ```properties

Reply:
I think I already made this change

Reply:
Oh, I think I missed one. Made the change now.